test/__snapshots__/build-css.jest.js.snap

```diff
@@ -1437,11 +1437,16 @@ textarea{
 .z4{ z-index:4 !important; }
 .z5{ z-index:5 !important; }
 .mx-auto{ margin-left:auto !important; margin-right:auto !important; }
-.viewport-third{ height:33.3333vh !important; }
-.viewport-half{ height:50vh !important; }
-.viewport-twothirds{ height:66.6666vh !important; }
-.viewport-almost{ height:90vh !important; }
-.viewport-full{ height:100vh !important; }
+.h-viewport-1\\\\/3{ height:33.3333vh !important; }
+.h-viewport-1\\\\/2{ height:50vh !important; }
+.h-viewport-2\\\\/3{ height:66.6666vh !important; }
+.h-viewport-9\\\\/10{ height:90vh !important; }
+.h-viewport-full{ height:100vh !important; }
+.hmax-viewport-1\\\\/3{ max-height:33.3333vh !important; }
+.hmax-viewport-1\\\\/2{ max-height:50vh !important; }
+.hmax-viewport-2\\\\/3{ max-height:66.6666vh !important; }
+.hmax-viewport-9\\\\/10{ max-height:90vh !important; }
+.hmax-viewport-full{ max-height:100vh !important; }
```

Comment:
Hmm the extra escaping characters looks wrong here

Reply:
@tristen think there are a few layers of escapes going on (manual escape in CSS, then another couple escapes applied by the snapshot code in Jest) but I am pretty sure this is working correctly.